### PR TITLE
SW-3849 Add latitude and longitude to accession CSV

### PR DESCRIPTION
**Pull request by sgrimm**
Allow users to specify the coordinates where seeds were collected when they import
accessions from a CSV file.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1280** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1280" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1280?utm_source=stack-comment).

File: src/main/resources/csv/accessions-template_es.csv [hunk 1]
```diff
@@ -30,4 +30,8 @@ Elija de la siguiente lista o se producirá un error durante la carga:
 - Salvaje
 - Reintroducido
 - Cultivado
-- Otros",Número de plantas,ID de planta
\ No newline at end of file
+- Otros",Número de plantas,ID de planta,"Latitud
+
+Latitud decimal del lugar de recolección de semillas (positivo = norte, negativo = sur)","Longitud
+
+Longitud decimal del lugar de recolección de semillas (positivo = este, negativo = oeste)"
```
Comment (karthikbtf) [inline src/main/resources/csv/accessions-template_es.csv:37]: just curious how this translation was obtained (phrase UI vs google etc?)

**Review by karthikbtf — APPROVED** [review]
👍

File: src/main/resources/csv/accessions-template_es.csv (hunk 1 above)
Comment (sgrimm) [reply src/main/resources/csv/accessions-template_es.csv:37]: Google Translate; we don't have Phrase set up to do translations of files at this point.

**Pull request merged**